test/redirects.ts

```diff
@@ -81,6 +81,20 @@ test('throws on endless redirects', withServer, async (t, server, got) => {
 	t.deepEqual(error.response.redirectUrls, new Array(10).fill(`${server.url}/`));
 });
```

Comment:
Consider renaming the original test (above) to indicate it is now checking for _default_ behavior.

Reply:
Forgot about it, will do.